Makefile

```diff
@@ -95,11 +95,18 @@ serve: .state/docker-build
 debug: .state/docker-build
 	docker-compose run --rm --service-ports web
 
-tests:
+tests_backend:
 	docker-compose run --rm web env -i ENCODING="C.UTF-8" \
 								  PATH="/opt/warehouse/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" \
 								  bin/tests --postgresql-host db $(T) $(TESTARGS)
 
+tests_frontend:
```

Comment:
Perhaps these would be better as `web_tests` and `static_tests`? Since the differentiation is which container they're being run in.